aspnet/tutorials/first-mvc-app/adding-view.rst

```diff
@@ -78,19 +78,18 @@ Open *Views/HelloWorld/Index.cshtml*. There are two places to make a change:
 You'll make them slightly different so you can see which bit of code changes which part of the app.
 
 .. literalinclude:: start-mvc/sample/src/MvcMovie/Views/HelloWorld/Index2.cshtml
-  :language: HTML
-  :linenos: 
-  :emphasize-lines: 2, 5
-
-Line number 2 in the code above sets the ``Title`` property of the `ViewDataDictionary <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewFeatures/ViewDataDictionary/index.html>`__ to "Movie List". The ``Title`` property is used in the ``<title>`` HTML element in the layout page:
+	:language: HTML
+	:emphasize-lines: 2, 5
+	
+``ViewData["Title"] = "Movie List";`` in the code above sets the ``Title`` property of the `ViewDataDictionary <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewFeatures/ViewDataDictionary/index.html>`__ to "Movie List". The ``Title`` property is used in the ``<title>`` HTML element in the layout page:
 
 .. code-block:: HTML
 
-  <title>@ViewData["Title"] - Movie App</title>
+	<title>@ViewData["Title"] - Movie App</title>
 
 Save your change and refresh the page. Notice that the browser title, the primary heading, and the secondary headings have changed. (If you don't see changes in the browser, you might be viewing cached content. Press Ctrl+F5 in your browser to force the response from the server to be loaded.) The browser title is created with ``ViewData["Title"]`` we set in the **Index.cshtml** view template and the additional "- Movie App" added in the layout file.
 
-Also notice how the content in the *Index.cshtml* view template was merged with the *Views/Shared/_Layout.cshtml* view template and a single HTML response was sent to the browser. Layout templates make it really easy to make changes that apply across all of the pages in your application.
+Also notice how the content in the *Index.cshtml* view template was merged with the *Views/Shared/_Layout.cshtml* view template and a single HTML response was sent to the browser. Layout templates make it really easy to make changes that apply across all of the pages in your application. You can read about the :doc:`/mvc/views/layout` file :doc:`here </mvc/views/layout>`.
```

Comment:
> You can read about the :doc:`/mvc/views/layout` file :doc:`here </mvc/views/layout>`

To avoid the double link how about `` To learn more see :doc:`/mvc/views/layout`. ``